File: Frontend/Model/Customer.py
class Customer:
    def __init__(self, customer_id, password, full_name, email, phone_number, is_manager=False):  # שים לב ל-m הקטנה
        self.customer_id = customer_id
        self.password = password
        self.full_name = full_name
        self.email = email
        self.phone_number = phone_number
        self.is_manager = is_manager  # שים לב ל-m הקטנה

    def __repr__(self):
        return (f"Customer(ID={self.customer_id}, "
                f"FullName='{self.full_name}', Email='{self.email}', PhoneNumber='{self.phone_number}'), IsManager='{self.is_manager}'")

File: Backend/server/Models/Customer.cs
using System;
using System.Collections.Generic;

namespace ServicesGatway.Models;

public partial class Customer
{
    public int CustomerId { get; set; }

    public string Password { get; set; } = null!;

    public string FullName { get; set; } = null!;

    public string Email { get; set; } = null!;

    public string PhoneNumber { get; set; } = null!;

    public bool IsManager { get; set; } = false;

    // public virtual ICollection<Ticket> Tickets { get; set; } = new List<Ticket>();
}
